Vijay-Kerala.md

```diff
@@ -0,0 +1,13 @@
+#Vijay , Kerala
+
+### Restaurants
+- The Indian Coffee house
+-Saravana Bhavan
+
+###Local Attractions
+
+-Science city
+-Vivekananda statue
+
+###For the nature
+-Pookot lake
```

Comment:
This sounds like a ✨  place, @vivekmanand -- what kinds of activities can you do there?